lib/src/signed_video_h26x_sign.c

```diff
@@ -119,8 +119,9 @@ complete_sei_nalu_and_add_to_prepend(signed_video_t *self)
   // Transfer oldest pointer in |payload_buffer| to local |payload|
```

Comment:
Since we touch `self->payload_buffer_idx` there should be a sanity check on it.
```
if (self->payload_buffer_idx < 1) return SVI_NOT_SUPPORTED;
```